codec_specs.md

```diff
@@ -561,16 +561,17 @@ Codec ID: A_AC3
 
 Codec Name: Dolby Digital AC-3
 
-Description: BSID <= 8 !! The private data is void ??? Corresponding ACM wFormatTag : 0x2000 ; channel number have
-to be read from the corresponding audio element
+Description: This corresponds to a bitstream with the value of the "Bit Stream Identification" `bsid` in the "Bit Stream Information" structure set to 8 as defined in Section 5.4.2.1 of [@!ATSC-A52] or Section 4.4.2.1 of [@!ETSI-TS102-366].
```

Comment:
I agree. Having the suffix of `/BSID<n>` is a historical accident (at least I would call it a mistake or an accident today); we should have just used `A_AC3` for any variant of AC-3 prior to E-AC-3, which rightfully gets its own `CodecID`.

At the moment mkvmerge still uses `A_AC3/BSID9` & `A_AC3/BSID10`, but I've wanted to change that for a long time now. Maybe I'll do it now.

Personally I prefer Jérôme's last variant as well.

Reply:
Thanks for the feedback!

Indeed ffmpeg does the [sampling rate division](https://github.com/FFmpeg/FFmpeg/blob/153a6dc8faafc4de263a493484ffc1dc2b5b26b2/libavcodec/ac3_parser.c#L129) as you mentioned. [VLC](https://code.videolan.org/videolan/vlc/-/blob/f1a90394435471e75fee6f13383079a3d0272ec4/modules/packetizer/a52.h#L204) as well.

For the bsid, VLC considers it's EAC3 if the [bsid is 11 to 16](https://code.videolan.org/videolan/vlc/-/blob/f1a90394435471e75fee6f13383079a3d0272ec4/modules/packetizer/a52.h#L409). 

10 or below seems fine for `A_AC3`. I'll use the NOT RECOMMEND for BSID9/BSID10 because neither libaformat nor VLC handle it anyway.

Reply:
> I'll use the NOT RECOMMEND for BSID9/BSID10 because neither libaformat nor VLC handle it anyway.

It would be a so easy update to do :-p.
(files exist so better to have it in most used tools...)

Reply:
I did not group BSID9/BSID10 because they are not well supported, IMO they should be marked as NOT RECOMMENDED.

Reply:
> It would be a so easy update to do :-p.

It is, except everyone using older versions will never benefit from this. And there's no good reason to use them in the future.